### PR TITLE
Adding a bit on error handling

### PR DESCRIPTION
**Pull request by mscharlock**

### CODE REVIEW

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/mscharlock-handleerrors/docs/excel/custom-functions-overview.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C8%5C21%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201808212225207564-335%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=vavexC29r9nvt0tmMcU%2BesFY%2F%2FVf80gL1FuMuCvBaOs%3D&st=2018-08-21T22%3A21%3A31Z&se=2018-09-21T22%3A26%3A31Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/excel/custom-functions-overview.md [hunk 1]
```diff
@@ -149,6 +149,25 @@ You can also use the following syntax:
...
 
+## Handling errors
+Error handling for custom functions is the same as [error handling for the Excel JavaScript API at large](https://docs.microsoft.com/en-us/office/dev/add-ins/excel/excel-add-ins-error-handling). Generally, you will use `.catch` to handle errors. The code below gives an example of `.catch()`. 
+
+```js
+function getComment(x) {
+    var url = "https://jsonplaceholder.typicode.com/comments/" + x; //this delivers a section of lorem ipsum from the jsonplaceholder API
+    return fetch(url)
+        .then(function (data) {
+            return data.json()
```
Comment (AlexJerabek) [inline docs/excel/custom-functions-overview.md]: Missing `;`

File: docs/excel/custom-functions-overview.md [hunk 2]
```diff
@@ -149,6 +149,25 @@ You can also use the following syntax:
 Office.Preview.StartCustomFunctions();
 ```
 
+## Handling errors
+Error handling for custom functions is the same as [error handling for the Excel JavaScript API at large](https://docs.microsoft.com/en-us/office/dev/add-ins/excel/excel-add-ins-error-handling). Generally, you will use `.catch` to handle errors. The code below gives an example of `.catch()`.
```
Comment (AlexJerabek) [inline docs/excel/custom-functions-overview.md]: Be consistent whether or not you use `()` when referring to catch.

Comment (AlexJerabek) [inline docs/excel/custom-functions-overview.md]: Change this link to be relative. Also, avoid including en-us in links, as it breaks localization.

**Comment by o365devx** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/mscharlock-handleerrors/docs/excel/custom-functions-overview.md) | :warning:Warning | | [Details](#user-content-docs%2Fexcel%2Fcustom-functions-overview.md)

<a name="docs%2Fexcel%2Fcustom-functions-overview.md"></a>
### [docs/excel/custom-functions-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/mscharlock-handleerrors/docs/excel/custom-functions-overview.md)
 - **Line 153**: **[Warning]** Invalid file link:(~/excel/excel-add-ins-error-handling).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C8%5C21%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201808212300083870-335%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=MS5Ghu0SYKtYP5IjyNjmWQghIqlqmuqtFt3yM1INEBo%3D&st=2018-08-21T22%3A56%3A19Z&se=2018-09-21T23%3A01%3A19Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/mscharlock-handleerrors/docs/excel/custom-functions-overview.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C8%5C21%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201808212302381890-335%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=bNXAqlyJEfvmV8X4jBLJM8jvKtv3Y1CrAKXrT7qI5mA%3D&st=2018-08-21T22%3A58%3A43Z&se=2018-09-21T23%3A03%3A43Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**